app/src/main/java/com/example/bigowlapp/model/Notification.java

```diff
@@ -115,6 +130,13 @@ public long timeSinceCreationMillis() {
         return Timestamp.now().toDate().getTime() - this.getCreationTime().toDate().getTime();
     }
 
+    public String getTitle(){
```

Comment:
Should have the @Exclude annotation since we don't want to push this variable in the database.

Reply:
Done.